.flake8

```diff
@@ -0,0 +1,32 @@
+[flake8]
+
+ignore =
+    # defaults flake8 ignores
+    E121,E123,E126,E226,E24,E704,W503,W504
+    # Function name should be lowercase
+    N802
+    # lowercase ... imported as non lowercase
+    # Useful to ignore for "import keras.backend as K"
+    N812
+    # TODO: remove all the exceptions below
+    # do not use bare 'except'
+    E722
+    # Imported but unused
+    F401
+    # the backslash is redundant between brackets
+    E502
+    # do not assign a lambda expression, use a def
+    E731
+    # local variable is assigned to but never used
+    F841
+    # trailing whitespace
+    W291
+    # expected 2 blank lines after class or function definition, found 1
+    E305
+
+
+#imported but unused in __init__.py, that's ok.
+per-file-ignores = **/__init__.py:F401
+
+# TODO: reduce the max line length to 90-ish
+max-line-length = 210
```

Comment:
Rationale for 210? Since we're going with black should we use their default 88?

Reply:
Some lines are too long in this repo. I didn't want to make them shorter in this PR to avoid having a PR with too many diffs.

I'll change it in the same PR applying black to the repo. Black will make them shorter for me. I won't have to do it manually.